.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,43 @@
+name: IntegrationTest
+# Need GitHub secret: DOCKER_HUB_USER, DOCKER_HUB_SECRETS, GHCR_TOKEN
+
+on:
+  push:
+    branches:
+      - master
+      - tekton-support
+      - jenkins-client-refactor
+      - feat/*
+      - fix/*
+      - test-* # make it be easier for contributors to test
+    tags:
+      - 'v*.*.*'
+  pull_request:
+    branches:
+      - 'master'
+      - 'tekton-support'
+      - 'jenkins-client-refactor'
+      - 'feat/*'
+      - 'fix/*'
+
+jobs:
+  IntegrationTest:
```

Comment:
This job name may need revise as `TektonBackend`.

Reply:
IMO, it is not necessary

Reply:
ok